src/checkReactVersionAtLeast.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+export function checkReactVersionAtLeast(
```

Comment:
Can we rename file to `react-version.ts` or something simplier then the current name?